README.md

```diff
@@ -41,8 +41,12 @@ We appreciate sample model contributions; they help ensure a consistent glTF eco
 
 To contribute a model, open a pull request with:
 * A new subdirectory containing
-   * The model in as many glTF variations as possible (using the same directory structure as the others ([example](1.0/Box))).  Tools for converting to glTF are [here](https://github.com/KhronosGroup/glTF#converters).
-   * A README.md with any usage restrictions.
+   * The model in as many glTF variations as possible (using the same directory structure as the others ([example](2.0/Box))).  Tools for converting to glTF are [here](https://github.com/KhronosGroup/glTF#converters).
+   * A screenshot of the model, stored in a subdirectory called `/screenshot`
+   * A README.md with information about the model. As shown in [this example](https://raw.githubusercontent.com/javagl/glTF-Sample-Models/master/2.0/Box/README.md), this file should at least include the following elements:
```

Comment:
Could a link to `README.md` be relative here?

Reply:
The intention was to make it a *raw* link, so that people can easily Copy+Paste the contents of the README.md that it links to (e.g. the image link, with this magic `!`), without having to manually go to the "raw" view. If it is OK that users will have to manually select the "raw" view, I can change it.

Reply:
I see. In such case, could the link point to the original KhronosGroup repo?

Reply:
Sure, thanks, I overlooked that. Fixed now.